### PR TITLE
Update match header

### LINKED ISSUE #124: match header
Reminder for @bguzryanto to reopen the PR:

https://github.com/odota/ui/pull/114

### PR DESCRIPTION
**Pull request by bguzryanto**
fixes #124 

Not ready for merge. We need to change actions link.

### CODE REVIEW

**Comment by howardchung** [comment]
What actions link?

**Comment by bguzryanto** [comment]
@howardchung So I bit confused with Parse link, should it be '/request/matchid' or 'request#matchid' regarding current implementation use 'request#matchid'?

**Comment by howardchung** [comment]
It should be the hash

File: components/Match/MatchHeader.jsx [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+      <MatchActions match={match} />
+      <MatchMetadata match={match} />
+      <div className={matchStyle.Clear} />
+    </div>
+  </div>
+);
+
+const MatchActions = ({ match }) => (
+  <div className={matchStyle.MatchActionWrapper}>
+    <Link to={`/request/${match.match_id}`}> {/* TODO: Replace with true link */}
```
Comment (howardchung) [inline components/Match/MatchHeader.jsx]: Just request#${match.match_id}

File: components/Match/MatchHeader.jsx [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+      <FlatButton label="Parse Replay" secondary />
+    </Link>
+    <Link to="/download"> {/* TODO: Replace with true link */}
```
Comment (howardchung) [inline components/Match/MatchHeader.jsx]: match.replay_url

File: components/Match/MatchHeader.jsx [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+      <FlatButton label="Download Replay" secondary />
+    </Link>
+    <Link to="/jist.tv"> {/* TODO: Replace with true link */}
```
Comment (howardchung) [inline components/Match/MatchHeader.jsx]: port this from existing view

File: components/Match/MatchHeader.jsx [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+      <FlatButton label="Jist TV" secondary />
+    </Link>
+    <Link to="/dotacoach"> {/* TODO: Replace with true link */}
```
Comment (howardchung) [inline components/Match/MatchHeader.jsx]: port from existing view

File: components/Match/MatchHeader.jsx [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+  </div>
+);
+
+const MatchMetadata = ({ match }) => (
+  <table className={matchStyle.Table}>
+    <thead>
+      <tr>
+        <th className={matchStyle.TableCell}>MatchID</th>
+        <th className={matchStyle.TableCell}>Mode</th>
+        <th className={matchStyle.TableCell}>Bracket</th>
```
Comment (howardchung) [inline components/Match/MatchHeader.jsx]: Skill

File: components/Match/MatchHeader.jsx [hunk 6]
```diff
@@ -0,0 +1,68 @@
...
+      <FlatButton label="DotaCoach" secondary />
+    </Link>
+  </div>
+);
+
+const MatchMetadata = ({ match }) => (
+  <table className={matchStyle.Table}>
+    <thead>
+      <tr>
+        <th className={matchStyle.TableCell}>MatchID</th>
```
Comment (howardchung) [inline components/Match/MatchHeader.jsx]: Match ID

**Comment by bguzryanto** [comment]
Ok, Will work on this soon. I have busy week :(

File: components/Match/MatchHeader.jsx [hunk 7]
```diff
@@ -0,0 +1,118 @@
...
+  <a
+    className={matchStyle.ActionButton}
+    rel="noopener noreferrer"
+    target="_blank"
+    href={`https://www.jist.tv/create.php?dota2-match-url=${replayUrl}`}
+  >
+    <span className={matchStyle.JistTvText}>Get Video with</span>
+    <img
+      className={matchStyle.JistTvImg}
+      src="https://www.opendota.com/public/images/jist-white-logo.png"
```
Comment (howardchung) [inline components/Match/MatchHeader.jsx]: copy the image over to assets directory.

File: components/Match/MatchHeader.jsx [hunk 8]
```diff
@@ -0,0 +1,118 @@
...
+  <a
+    target="_blank"
+    rel="noopener noreferrer"
+    className={matchStyle.ActionButton}
+    href={`https://dotacoach.org/Hire/Yasp?matchID=${matchId}&userSteamId=${steamId}&playerMmr=${playerMmr}`}
+  >
+    <span className={matchStyle.DotaCoachText}>Ask a coach</span>
+    <img
+      className={matchStyle.DotaCoachImg}
+      src="https://www.opendota.com/public/images/dotacoach-32x24.png"
```
Comment (howardchung) [inline components/Match/MatchHeader.jsx]: copy the image over

File: components/Match/MatchHeader.jsx [hunk 9]
```diff
@@ -0,0 +1,118 @@
...
+  <a className={matchStyle.ActionButton} href={replayUrl}>
+    Download Replay
+  </a>
+);
+
+const MatchActions = ({ match, user }) => {
+  let steamId = '';
+  let playerMmr = '';
+
+  if (typeof(user) === 'object' && 'steam_id' in user) {
```
Comment (howardchung) [inline components/Match/MatchHeader.jsx]: less verbose: `user && user.steam_id`

**Pull request merged**